go/state/gostate/archive_state.go

```diff
@@ -76,7 +76,9 @@ func (s *ArchiveState) GetStorage(address common.Address, key common.Key) (commo
 	}
 	return storage, s.archiveError
 }
-
+func (s *ArchiveState) HasEmptyStorage(common.Address) (bool, error) {
+	panic("HasEmptyStorage: not implemented for ArchiveState")
```

Comment:
As for the C++ case, it might be that this is breaking nightly builds of Carmen on LaScala. We should apply the same (temporary) fix here (and at other places) as for the C++ implementation.